movie/views.py

```diff
@@ -14,6 +14,13 @@ def index(request):
     random_movie = movie.order_by("?")[:30]
     reviews = Review.objects.select_related("movie").order_by("-created_at")[:10]
 
+    recommended_movies = None
+    if request.user.is_authenticated:
+        top_genres = Genre.objects.top_reviewed_by_user(request.user)
+        recommended_movies = Movie.objects.recommended_by_user(
+            user=request.user, genres=top_genres, limit=12
```

Comment:
```suggestion
            user=request.user, genres=top_genres, limit=15
```
15개로 해야 carousel 개수가 맞을거 같아요